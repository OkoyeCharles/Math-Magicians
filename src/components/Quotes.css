.quotes {
  padding: 2em;
}

blockquote cite {
  color: rgba(0, 0, 0, 0.485);
  font-size: 0.85em;
}

@media screen and (min-width: 800px) {
  .quotes {
    padding: 5em;
    background-image: url(./images/think.svg);
    background-position: 100% 0%;
    background-repeat: no-repeat;
    background-size: 50vw;
    height: 100vh;
  }

  blockquote {
    font-size: 1.5em;
    width: 30%;
  }
}
