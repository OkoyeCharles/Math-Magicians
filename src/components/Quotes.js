import React from 'react';
import './Quotes.css';

function Quotes() {
  return (
    <div className="quotes">
      <blockquote>
        Doing a mathematical calculation in your head, not trustiing your
        answer, so you ended up using a calculator.
        <br />
        <cite>&quot;Anonymous&quot;</cite>
      </blockquote>
    </div>
  );
}

export default Quotes;
