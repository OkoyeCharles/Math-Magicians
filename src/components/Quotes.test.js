import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';

describe('correctlly renders a component', () => {
  test('should render quote component', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
