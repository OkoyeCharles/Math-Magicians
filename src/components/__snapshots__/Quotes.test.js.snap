// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctlly renders a component should render quote component 1`] = `
<div
  className="quotes"
>
  <blockquote>
    Doing a mathematical calculation in your head, not trustiing your answer, so you ended up using a calculator.
    <br />
    <cite>
      "Anonymous"
    </cite>
  </blockquote>
</div>
`;
